test/Main.purs

```diff
@@ -5,7 +5,7 @@ module Test.Main where
 
   import Optic.Core
 
-  import Prelude (class Show, Unit, (#), ($), (+), (<>), bind, const, show)
+  import Prelude (class Show, Unit, (#), ($), (+), (<>), const, show, discard)
```

Comment:
Fixes this error:

    Unknown value discard. You're probably using do-notation, which the compiler replaces with calls to the discard function. Please import discard from module Prelude

See `Implicitly discarded values in do blocks now raise errors` in https://github.com/purescript/purescript/releases/tag/v0.11.0